tasks/configure.yml

```diff
@@ -50,6 +51,7 @@
 - name: Check for OpenVPN iptable rule
   become: yes
   command: "iptables -t nat -C POSTROUTING -s {{ openvpn.subnet }} -o eth0 -j MASQUERADE"
+  changed_when: false
```

Comment:
Could you explain why you are making this addition pls?

Reply:
This task always runs, without this the idempotency check fails.

Reply:
Thats what I was thinking.. 
It will create an extra entry in the table every time it runs, however.
This is legacy code, but for future use of iptables, we should prefer the ansible iptables module.

Reply:
Defo, I will pop a ticket in for this.